### PR TITLE
Remove old incompatible option --incompatible_symlinked_sandbox_expands_tree_artifacts_in_runfiles_tree.

### PR DESCRIPTION
**Pull request by benjaminp**
This one was flipped more than a year ago.

RELNOTES: Remove old incompatible flag --incompatible_symlinked_sandbox_expands_tree_artifacts_in_runfiles_tree.

### CODE REVIEW

File: src/main/java/com/google/devtools/build/lib/sandbox/DarwinSandboxedSpawnRunner.java [hunk 1]
```diff
@@ -232,11 +232,7 @@ protected SandboxedSpawn prepareSpawn(Spawn spawn, SpawnExecutionContext context
 
     SandboxInputs inputs =
         helpers.processInputFiles(
-            context.getInputMapping(
-                getSandboxOptions().symlinkedSandboxExpandsTreeArtifactsInRunfilesTree),
-            spawn,
-            context.getArtifactExpander(),
-            execRoot);
+            context.getInputMapping(true), spawn, context.getArtifactExpander(), execRoot);
```
Comment (meisterT) [inline src/main/java/com/google/devtools/build/lib/sandbox/DarwinSandboxedSpawnRunner.java:235]: Do we call `getInputMapping` with `false` anywhere?

Comment (benjaminp) [reply src/main/java/com/google/devtools/build/lib/sandbox/DarwinSandboxedSpawnRunner.java:235]: Not as far as I can see in Bazel. I was hoping to follow up with a CL removing that parameter if someone could assure me that Google is not passing `false`.

Comment (meisterT) [reply src/main/java/com/google/devtools/build/lib/sandbox/DarwinSandboxedSpawnRunner.java:235]: Ok, let's do that in a separate CL then.

Comment (meisterT) [reply src/main/java/com/google/devtools/build/lib/sandbox/DarwinSandboxedSpawnRunner.java:235]: https://bazel-review.googlesource.com/c/bazel/+/137717

Comment (benjaminp) [reply src/main/java/com/google/devtools/build/lib/sandbox/DarwinSandboxedSpawnRunner.java:235]: Thanks—I also raced and submitted #11809.

Comment (meisterT) [reply src/main/java/com/google/devtools/build/lib/sandbox/DarwinSandboxedSpawnRunner.java:235]: Sorry for the duplicate work, I thought it's easier for me to do with the internal changes that were necessary.

Comment (benjaminp) [reply src/main/java/com/google/devtools/build/lib/sandbox/DarwinSandboxedSpawnRunner.java:235]: No problem; it was almost completely mechanical anyway.

**Pull request closed without merging**